qdev_wrappers/fitting/Fitclasses.py

```diff
@@ -0,0 +1,72 @@
+
+import qcodes as qc
+import numpy as np
+import scipy.fftpack as fftpack
+
+
+class Fitclass:
```

Comment:
Would also make sense for this class to have the functions 'fun' and 'guess' which throw the NotImplementedError in the base class and then are implemented in the classes that inherit. This is also maybe a good place for us to say we are doing a least squares fit and then make this a LeastSqauresFit class. This of course only holds if we are only going to use this class together with the 'fitter' (function or class, it's currently a class but we talked about turning it into a function), at the moment seems reasonable to me but perhaps it's a bit weird to specify that here when there is no actual 'least squares fit' going on in this class.